Day 1 Khan/SQL advanced

```diff
@@ -0,0 +1,218 @@
+INSERT INTO songs (title, artist, mood, duration, released)
+    VALUES ("Sweet Caroline", "Neil Diamond", "happy", 201, 1969);
+INSERT INTO songs (title, artist, mood, duration, released)
+    VALUES ("Call me maybe", "Carly Rae Jepsen", "happy", 193, 2011);
+*/
+
+    SELECT title FROM songs;
+SELECT title FROM songs WHERE mood LIKE "epic" OR released > 1990;
+SELECT title FROM songs WHERE mood LIKE "epic" AND released > 1990 AND duration < 240;
+
+/* Challenge: Playlist maker
+
+Now you'll make a 'Pop' playlist. In preparation, select the name of all of the artists from the 'Pop' genre.
+(Tip: Make sure you type it 'Pop', SQL considers that different from 'pop'.)
+*/
+/*
+CREATE TABLE artists (
+    id INTEGER PRIMARY KEY AUTOINCREMENT,
+    name TEXT,
+    country TEXT,
+    genre TEXT);
+
+INSERT INTO artists (name, country, genre)
+    VALUES ("Taylor Swift", "US", "Pop");
+INSERT INTO artists (name, country, genre)
+    VALUES ("Led Zeppelin", "US", "Hard rock");
+INSERT INTO artists (name, country, genre)
+    VALUES ("ABBA", "Sweden", "Disco");
+INSERT INTO artists (name, country, genre)
+    VALUES ("Queen", "UK", "Rock");
+INSERT INTO artists (name, country, genre)
+    VALUES ("Celine Dion", "Canada", "Pop");
+INSERT INTO artists (name, country, genre)
+    VALUES ("Meatloaf", "US", "Hard rock");
+INSERT INTO artists (name, country, genre)
+    VALUES ("Garth Brooks", "US", "Country");
+INSERT INTO artists (name, country, genre)
+    VALUES ("Shania Twain", "Canada", "Country");
+INSERT INTO artists (name, country, genre)
+    VALUES ("Rihanna", "US", "Pop");
+INSERT INTO artists (name, country, genre)
+    VALUES ("Guns N' Roses", "US", "Hard rock");
+INSERT INTO artists (name, country, genre)
+    VALUES ("Gloria Estefan", "US", "Pop");
+INSERT INTO artists (name, country, genre)
+    VALUES ("Bob Marley", "Jamaica", "Reggae");
+
+CREATE TABLE songs (
+    id INTEGER PRIMARY KEY AUTOINCREMENT,
+    artist TEXT,
+    title TEXT);
+
+INSERT INTO songs (artist, title)
+    VALUES ("Taylor Swift", "Shake it off");
+INSERT INTO songs (artist, title)
+    VALUES ("Rihanna", "Stay");
+INSERT INTO songs (artist, title)
+    VALUES ("Celine Dion", "My heart will go on");
+INSERT INTO songs (artist, title)
+    VALUES ("Celine Dion", "A new day has come");
+INSERT INTO songs (artist, title)
+    VALUES ("Shania Twain", "Party for two");
+INSERT INTO songs (artist, title)
+    VALUES ("Gloria Estefan", "Conga");
+INSERT INTO songs (artist, title)
+    VALUES ("Led Zeppelin", "Stairway to heaven");
+INSERT INTO songs (artist, title)
+    VALUES ("ABBA", "Mamma mia");
+INSERT INTO songs (artist, title)
+    VALUES ("Queen", "Bicycle Race");
+INSERT INTO songs (artist, title)
+    VALUES ("Queen", "Bohemian Rhapsody");
+INSERT INTO songs (artist, title)
+    VALUES ("Guns N' Roses", "Don't cry");
+    */
+
+SELECT title FROM songs WHERE artist = "Queen";
+SELECT name FROM artists WHERE genre LIKE "pop";
+Select title From songs Where artist IN (
```

Comment:
please capitalize `SELECT`, `FROM`, and `WHERE`.

Also, I prefer the consistency of either using `=` like you have on line 103 or always using `LIKE` like you've got on line 104. I'd err on the side of using `=` because that is the most accurate for what you're trying to do since you're not looking for items that are `LIKE`, but that are exactly. 

Also, capitalize `"Pop"` on line 104 since that is the exact entry from the database.